rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/values.kt

```diff
@@ -1038,17 +1038,18 @@ fun areEquals(value1: Value, value2: Value): Boolean {
 
 @Serializable
 object VoidValue : Value {
-
-    override fun asString(): StringValue {
-        TODO("Not yet implemented")
+    override fun toString(): String {
+        return "VoidValue"
     }
 
     override fun assignableTo(expectedType: Type): Boolean {
-        TODO("Not yet implemented")
```

Comment:
Depends, if we consider VoidValue as a null object we can assign it to every variable, it's a method to set to null a typed variable. Perhaps the right solution is to return to NullValue and leave VoidValue unchanged because VoidValue is an empty value, not a null one.